pom.xml

```diff
@@ -210,197 +210,199 @@
+                            </sourceFileIncludes>
+                            <quiet>true</quiet>
+                            <windowtitle>GeoWave ${project.version}</windowtitle>
+                            <doctitle>GeoWave ${project.version}</doctitle>
+                            <groups>
+                                <group>
+                                    <title>Analytics</title>
+                                    <packages>mil.nga.giat.geowave.analytics*</packages>
+                                </group>
+                                <group>
+                                    <title>GeoTools Storage</title>
+                                    <packages>mil.nga.giat.geowave.gt*</packages>
+                                </group>
+                                <group>
+                                    <title>Storage</title>
+                                    <packages>mil.nga.giat.geowave.accumulo*:mil.nga.giat.geowave.store*</packages>
+                                </group>
+                                <group>
+                                    <title>Indexing</title>
+                                    <packages>mil.nga.giat.geowave.index*</packages>
+                                </group>
+                                <group>
+                                    <title>Ingest Framework</title>
+                                    <packages>mil.nga.giat.geowave.ingest*</packages>
+                                </group>
+                                <group>
+                                    <title>Base Ingest Types</title>
+                                    <packages>mil.nga.giat.geowave.types*</packages>
+                                </group>
+                            </groups>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.eluder.coveralls</groupId>
+                <artifactId>coveralls-maven-plugin</artifactId>
+                <version>3.1.0-munge-hack</version>
+                <configuration>
+                    <jacocoReports>
+                        <jacocoReport>${project.basedir}/geowave-index/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-analytics/target/site/jacoco-ut/jacoco.xml
+                        </jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-ingest/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-raster/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-store/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-types/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        -
```

Comment:
Do we need this '-'? Can I remove it.  If so, I have the rebased  version ready to push and then merge.